### PR TITLE
Ecooper/cvu 2227

### PR DESCRIPTION
**Pull request by cooperevelyn**
News API PR

### CODE REVIEW

File: src/app/api.service.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    console.log(endpoint_url);
+    console.log(this.http.get(endpoint_url)['articles']);
+    console.log(this.http.get(endpoint_url));
+
+
+    return this.http.get(this.url);
+    // return this.http.get(`https://newsapi.org/v2/top-headlines?' + 'country=us' + '&apiKey=${this.API_KEY}`);
+  }
+
+  public getTr(text: string) {
```
Comment (ScottFSeely) [inline src/app/api.service.ts:32]: No need to change but in general be more verbose with variable and function names... `getTranslation()`

File: src/app/first/first.component.html [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+
+    <div class="btn-group" role="group" aria-label="Basic example">
+      <button type="button" class="btn btn-secondary">first menu</button>
+      <button type="button" class="btn btn-secondary">second menu</button>
+      <button type="button" class="btn btn-secondary">third menu</button>
+      <button type="button" class="btn btn-secondary">fourth menu</button>
+    </div>
+  </div>
+  <div class="container-1">
+    <h1>Flexbox?</h1>
```
Comment (ScottFSeely) [inline src/app/first/first.component.html:14]: flexbox is great... I refer to this all the time [css-tricks](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

Another great layout helper is the Grid layout from [bootstrap/coreui](https://coreui.io/docs/2.1/layout/grid/)

File: src/app/gtranslate/gtranslate.component.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  ) { }
+
+  ngOnInit(): void {
+    console.log("subscribing");
+    this.subscription = this.apiService.getTr(this.words).subscribe(() => {});
+  }
+
+  ngOnDestroy(): void {
+    this.subscription.unsubscribe();
+    console.log(this.subscription.unsubscribe());
```
Comment (ScottFSeely) [inline src/app/gtranslate/gtranslate.component.ts:25]: unsubscribing twice here

File: src/app/news/news.component.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            <br/>
+            <div class="box shift">
+                <button class="btn btn-pill btn-warning" (click)="browseToggle()">{{buttonText}}</button>
+            </div>
+            <br/>
+        </div>
+        
+        <!-- <button (click)="getArticle()">Go</button>  -->
+
+        <div *ngIf="browse; else doSearch">
```
Comment (ScottFSeely) [inline src/app/news/news.component.html:16]: Nice!

File: src/app/gtranslate/gtranslate.component.ts (hunk 3 above)
Comment (cooperevelyn) [reply src/app/gtranslate/gtranslate.component.ts:25]: your're right. I'll delete one of them.

**Pull request merged**